test/Common.cpp

```diff
@@ -207,6 +207,74 @@ bool CommonOptions::parse(int argc, char const* const* argv)
 	return true;
 }
 
+template<typename T> std::string CommonOptions::toString(T& _filter) const
+{
+	std::stringstream options;
+	if(!m_singleton)
+		return "";
+
+	for(std::string& opt : _filter)
+	{
+		switch (options) {
+			case "evmVersion":
+				options << "evmVersion=" << evmVersion().name() << " ";
```

Comment:
I think adding a functionality like this could be helpful in the future if we want to print other options... But I don't know if this is a probable scenario or if it is worth the effort... some inputs about this would be appreciated.